jsons/parameters.json

```diff
@@ -8881,6 +8881,7 @@
         "DynamicMods",
         "StaticMods"
       ],
+      "mzml2mgf_style_1": "modifictions",
```

Comment:
Has to do with the new ursgal version
Maybe this will change, but I needed this to get mzml2mgf running
I'd leave this open until we decide if we can solve this another way

Reply:
I see why you put it in, just ran into the problem while executing the msfragger example ^^ you don't see it when executing pymzml alone. 
I would rather prefer to update the code in ursgal, since as Stefan pointed out pymzml doesn't care about modifications in the first place. Hence in ursgal.unode going from
`        if "modifications" in uparams.keys():
            uparams.map_mods()`     
to something like:
`        if "modifications" in uparams["translations"]["all_params"].keys():
            uparams.map_mods()`
since this dict contains all params relevant only for the given node.